modules/phase_field/test/tests/phase_field_kernels/MatGradSquareCoupled.i

```diff
@@ -0,0 +1,124 @@
+#
+# Test the MatGradSquareCoupled kernel (which adds -L*v to the residual) for the case
```

Comment:
Please remove the statement `(which adds -L*v to the residual)`. Sorry I didn't notice this before!